### PR TITLE
Fix/new colours

### LINKED ISSUE #102: Update Tailwind colour config with new colours from Figma
https://www.figma.com/file/XFbhstkgQFz8ZefAU3w2p4/1.-Quantum-Design-System?type=design&node-id=5-59&mode=dev

- [ ] Change the config
- [ ] Document the change in Storybook

### PR DESCRIPTION
**Pull request by Ademsk1**
fixes #102 
Adds in dark colour mode to the alias colour set. Applied to the current colours used.

### CODE REVIEW

**Review by ergusto — CHANGES_REQUESTED** [review]
Changes look good, but it seems to have broken the Storybook colors.md page. We could maybe also split the sections in that page up to reflect the categorisation of colours in the new folder structure.

**Review by ergusto — CHANGES_REQUESTED** [review]
Quite a few suggestions/change requests I'm afraid but mostly small things. I think we will be able to neaten up the colors.md page a bit as there's quite a lot of duplication there and I think we might be able to display it in a more useful/digestible format, but let's leave that for the time being.

File: src/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -15,16 +15,16 @@ const checkboxVariants = cva([
     'border-border'
   ],
   [
-    'data-[state=checked]:border-blue-600',
-    'data-[state=checked]:bg-blue-600',
+    'data-[state=checked]:border-primary-DEFAULT-600',
```
[the remark this answers is not in the document]
Comment (Ademsk1) [reply src/components/Checkbox/index.tsx:18]: I have, they look good. I don't think we can apply DEFAULT to another object filled with colours to ignore it.

File: stories/Colors.mdx [hunk 2]
```diff
@@ -8,42 +10,53 @@ import { colors, default as aliasColors } from '@/colors.ts';
...
 
+<ColorWrapper colorPalette={aliasColors.button.secondary} />
+### Secondary Borders
+<ColorWrapper colorPalette={aliasColors.button.secondary.border} />
+### Tertiary
+<ColorWrapper colorPalette={aliasColors.button.tertiary} />
+### Success
+<ColorWrapper colorPalette={aliasColors.button.success} />
+### Danger
+<ColorWrapper colorPalette={aliasColors.button.danger} />
```
Comment (ergusto) [inline stories/Colors.mdx:62]: It looks like the `ColorWrapper` component isn't working for these three colour sets, it's rendering a blank colour swatch for `dark` where it isn't defined. 

Can we also simplify these success and danger colours and just have a single row for each?

**Review by ergusto — CHANGES_REQUESTED** [review]
One more minor thing then good to go.

**Review by ergusto — CHANGES_REQUESTED** [review]
One more minor thing then good to go.

**Review by ergusto — CHANGES_REQUESTED** [review]

**Review by ergusto — APPROVED** [review]

**Pull request merged**